src/main/java/org/reficio/p2/P2Mojo.java

```diff
@@ -133,6 +133,16 @@
     private boolean skipInvalidArtifacts;
 
     /**
+     * Skip not existing artifacts.
+     *
+     * <p>
+     * This flag controls if the processing should be continued anyway, if an artifact does not exist. It defaults to
+     * false to keep the old behavior (break on not existing artifacts).
+     */
```

Comment:
That's true. It's due to the fact that I can't catch the `ArtifactResolutionException` as documented inline. I'm not happy with this implementation. Can you explain how a checked exception can bubble up, but is not in the method signature?

I also considered `instanceof ArtifactResolutionException`, but this is rather dirty. 

I would prefer to nail it down on `ArtifactResolutionException`because now it may lead to silenced errors. Do you have an idea?